### ISSUE TITLE
Chaos Engine AGA - CD32. version

### ISSUE THREAD

**Issue by ReadyPlayer29** [amigavision/AmigaVision#188]
Could we have Chaos Engine AGA and Chaos engine AGA CD32 please??

Same but different but would be nice to see Chaos Engine complete

**Comment by limi** [comment]
Are there any substantial reasons (different levels, different music) other than the botched AGA re-coloring? I haven’t played the AGA version super far into the game, but I assume it has the same gameplay?

**Comment by ReadyPlayer29** [comment]
CD32 version without the audio on attract mode is not as good, least AGA has title music.

**Comment by limi** [comment]
Yeah, we generally go for the AGA version over CD32 when available — but in the case of Chaos Engine we went with the ECS version, as the AGA version seems like a lazy over-saturated get-the-money upgrade. But if there are other redeeming versions of the AGA version over ECS, I am all ears!

**Comment by ReadyPlayer29** [comment]
AGA version has amazing music for the title and there must be some other reframing features that I or no one else has currently thought of

Do it for the music

**Comment by limi** [comment]
Note to self: Music in the ECS and AGA versions is identical.

**Comment by limi** [comment]
Closing since there doesn’t seem to be any other substantial differences.
